core/openapi.yaml

```
@@ -1,29 +1,29 @@
 openapi: 3.0.1
 info:
   title: OSRD Core
-  description: 'OSRD backend api description.'
+  description: "OSRD backend api description."
   termsOfService: http://swagger.io/terms/
   license:
     name: GNU GPLv3
     url: https://www.gnu.org/licenses/gpl-3.0.fr.html
   version: 0.1.0
 servers:
-- url: http://gateway.dgexsol.fr/osrd/
-  description: Production server
-- url: http://gateway.staging.dgexsol.fr/osrd/
-  description: Staging server
+  - url: http://gateway.dgexsol.fr/osrd/
+    description: Production server
```

We don't have production server yet. Bu we have the dev one:

```suggestion
  - url: http://gateway.dev.dgexsol.fr/osrd/
    description: Dev server
```